ui/package.json

```diff
@@ -32,6 +32,7 @@
     "@radix-ui/react-tabs": "^1.1.0",
     "@radix-ui/react-toast": "^1.1.5",
     "@radix-ui/react-tooltip": "^1.0.7",
+    "@t3-oss/env-core": "^0.11.1",
```

Comment:
Could you shortly summarize the benefits of using T3 in the commit message?

Reply:
Commit message nit: There is no point in using `[1]` if you don't refer to it 😉 I propose to change this to:

```
T3 Env [1] ...

...

[1]: https://env.t3.gg/docs/core
```

So refer to the footnote in the first sentence, and drop "See " from the footnote.